pkg/body_extractor/grpc_payload.go

```diff
@@ -24,12 +28,24 @@ type GRPCPayloadCompressionFormat uint8
 const (
 	compressionNone GRPCPayloadCompressionFormat = 0
 	compressionMade GRPCPayloadCompressionFormat = 1
-	maxInt                                       = int(^uint(0) >> 1)
+
+	maxInt = int(^uint(0) >> 1)
```

Comment:
how are we using this?

Reply:
oh this is from earlier change, will remove it

Reply:
in grpc request parser
https://github.com/odpf/shield/blob/main/pkg/body_extractor/grpc_payload.go#L84